src/i18n/it/ui.ts

```diff
@@ -0,0 +1,119 @@
+export default {
+	'a11y.skipLink': 'Vai al Contenuto',
+	'a11y.sectionLink': 'Sezione intitolata',
+	'navbar.a11yTitle': 'Cima',
```

Comment:
To read "cima" in a site is weird to me, but I can't come up with a better suggestion

Reply:
That's the only word that came to my mind too...
I'll try and find a better solution :+1:

Reply:
As a replacement we could use "Intestazione" or "Barra di navigazione". They are both clear indicators of what it is and since it is meant to be consumed by screen readers we can prioritize clarity over brevity.

Personally I prefer "Barra di navigazione".

Reply:
"Barra di navigazione" sounds good to me! Great suggestion